indico/modules/events/abstracts/blueprint.py

```diff
@@ -49,6 +50,7 @@
 _bp.add_url_rule('/manage/abstracts/open', 'open_abstracts_call', RHOpenCFA, methods=('POST',))
 _bp.add_url_rule('/manage/abstracts/close', 'close_abstracts_call', RHCloseCFA, methods=('POST',))
 _bp.add_url_rule('/manage/abstracts/boa', 'manage_boa', RHManageBOA, methods=('GET', 'POST'))
+_bp.add_url_rule('/manage/abstracts/boa.pdf', 'export_boa', RHExportBOA)
```

Comment:
This is not only used in management, so it should be `/book-of-abstracts.pdf` (I wouldn't even put `/abstracts/` in the URL)